server/views/pages/overview/partials/educationAndTrainingFunctionalSkills.test.ts

```diff
@@ -0,0 +1,125 @@
+import * as fs from 'fs'
```

Comment:
This is not really a new test; its a rename of the previously committed `server/views/pages/overview/partials/educationAndTrainingTabContents.test.ts` (which is shown below as a deletion)